packages/backend/src/peripherals/database/migrations/013_forced_trade_offers.ts

```diff
@@ -0,0 +1,51 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+export async function up(knex: Knex) {
```

Comment:
Why two separate tables instead of one?

Reply:
It's easier to make it type-safe from my perspective. If it were one DB, half of the fields would be nullable. With this approach, all the necessary fields are staying required.

Reply:
As long as I agree with you in general, in this case it results in more database and application code where at the same time we validate the correctness of data at the application level already.